modules/4337/contracts/Safe4337Module.sol

```diff
@@ -211,6 +211,42 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
         operationHash = keccak256(operationData);
     }
 
+    /**
+     * @dev Checks if the signatures length is correct and does not contain addtional bytes.
+     * @param signatures signatures data
+     * @param threshold Indicates the number of iterations to perform in the loop.
+     * @return result True if length check passes, false otherwise.
+     */
+    function _checkSignatureLength(bytes calldata signatures, uint256 threshold) internal pure returns (bool result) {
+        uint256 offset = threshold * 0x41;
+
+        if (signatures.length < offset) return false;
+
+        result = true;
```

Comment:
nit: Maybe we shouldn't reuse this variable, it makes the code harder to understand IMO.

Reply:
here: https://github.com/safe-global/safe-modules/pull/453/commits/53345a672f35578cb0d320b28f5f891b42386e70